h/static/styles/topbar.scss

```diff
@@ -41,7 +41,7 @@
 
     & > *:first-child, & > *:last-child {
       line-height: 28px;
-      margin: 0 .72em;
+      margin-right: .72em;
```

Comment:
What I mean is that if you resize the view to respond to a smaller screen you'll see that it does line up. Make that always the case.